sled-hardware/src/underlay.rs

```diff
@@ -79,7 +74,9 @@ pub fn find_nics() -> Result<Vec<AddrObject>, Error> {
 /// For a real Gimlet, this should return the devices like `cxgbeN`. For a
 /// developer machine, or generally a non-Gimlet, this will return the
 /// VNICs we use to emulate those Chelsio links.
-pub fn find_chelsio_links() -> Result<Vec<PhysicalLink>, Error> {
+pub fn find_chelsio_links(
```

Comment:
Moving this from a heuristic over a statically defined set of names to explicit configuration.

In the multi-switch development and testing environments we're building, the data links representing the chelsios may be `ixgbe`, an actual `cxgbe`, or a simnet link.

Reply:
:100: